src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayMiscellaneousOptions.cs

```diff
@@ -55,5 +55,13 @@ public enum SymbolDisplayMiscellaneousOptions
         /// the special question mark syntax.
         /// </summary>
         ExpandNullable = 1 << 5,
+
+        /// <summary>
+        /// Suppresses minimization of the first visited symbol.
```

Comment:
I have to admit, it's not clear to me what "first visited symbol" means in this case as an external consumer of the API. The visitor is an implementation detail of this API, no? It seems the term shouldn't appear here then.